Casks/v/virtualdj.rb

```diff
@@ -0,0 +1,31 @@
+cask "virtualdj" do
+  version "2024.b8124"
+  sha256 "fddcdd538b4b1a8a70b7cba07f1f1303c73a387eeae3e7600eccdf96c9ec2e08"
+
+  url "https://download.virtualdj.com/install_virtualdj_#{version.dots_to_underscores}_mac.pkg"
```

Comment:
```suggestion
  url "https://download.virtualdj.com/install_virtualdj_#{version.csv.first}_b#{version.csv.second}_mac.pkg"
```

Update url to match new `version` format.